src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1071,9 +1073,10 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMTransaction> const& m)
             std::weak_ptr<PeerImp> weak = shared_from_this();
             app_.getJobQueue ().addJob (
                 jtTRANSACTION, "recvTransaction->checkTransaction",
-                [weak, flags, stx] (Job&) {
+                [weak, flags, checkSignature, stx] (Job&) {
```

Comment:
Not surprising.  VS doesn't really adhere to any particular C++ standard.  Instead it implements whatever features it can and throws them into its latest version.  But gcc and clang strictly implement C++11 and C++14 features, activated only under the `-std=c++11` / `-std=c++14` flags respectively.  It maintains these as two distinct languages forever.